package.json

```diff
@@ -13,6 +13,7 @@
   "license": "Apache-2.0",
   "gypfile": true,
   "dependencies": {
+    "always-tail": "^0.2.0",
```

Comment:
There was talk about removing it from on-core. So it was initially add here. I can remove it though.